### ISSUE TITLE
get_dataloaders() param : is_local=args.local ERROR

### ISSUE THREAD

**Issue by MissPearson** [ChrisWu1997/MURA#1]
train_loader = get_dataloaders('train', batch_size=args.batch_size, shuffle=True, **is_local=args.local**)
is_local is not the param in the get_dataloaders.
